docs/Tutorials - start here/yolov5.md

```diff
@@ -0,0 +1,52 @@
+# How to Use Innofw Framework for Object Detection using YoloV5
+
+In this tutorial, we will show you how to use Innofw framework for object detection using YoloV5. We will show you how to create a configuration file for your dataset, how to create a configuration file for your training, how to train your model, how to evaluate your model and how to make an inference on your model.
+
+There are different variations of yolov5 library. For example, you can use yolov5s, yolov5m, yolov5l or yolov5x. The letter at the end of the name indicates the size of the model. The bigger the letter, the bigger the model. The bigger the model, the more accurate it is. However, the bigger the model, the slower it is. In this tutorial, we will use yolov5s. You can change the model by changing the `arch` parameter in the file `innofw\config\models\detection\yolov5.yaml`.
+
+## Dataset
+* You need to create a configuration file for your dataset and save it inside the following directory: `innofw\config\datasets\detection`. You can see the current configuration files that currently exist in the directory. You can use them as a template for your dataset. 
+
+* The configuration file should contain the following :
+    1- Link to your dataset (if it is public) or the path to your dataset (if it is private). You will need to give the link to the training set, testing set and inference set separatly
```

Comment:
Link to your dataset (if it is hosted on s3) or the path to your dataset (if it is on local machine)